src/components/atoms/ConversationMessage.vue

```diff
@@ -32,7 +32,6 @@
       v-if="isLastMessage"
       ref="chatReaderIcon"
       :src="icons[senderIcon]"
-      :alt="senderIconAltText"
```

Comment:
I'm not 100 on this but I believe we want it to have an empty alt tage ie alt="" similar to what you have below